### PR TITLE
Data srcs setup articles

### PR DESCRIPTION
**Pull request by afoonalex**

### CODE REVIEW

File: user-guide/SUMMARY.md [hunk 1]
```diff
@@ -62,6 +63,10 @@
 
 ## Data Sources Setup
 * [Amazon Athena Setup](data-sources/amazon-athena.md)
+* [BigQuery Setup](data-sources/bigquery.md)
+* [Google Analytics Instructions](data-sources/google-analytics.md)
```
Comment (arikfr) [inline user-guide/SUMMARY.md]: Google Analytics Setup

File: user-guide/about/supported_data_sources.md [hunk 2]
```diff
@@ -1,26 +1,36 @@
...
-- Impala
-- InfluxDB
-- Presto
-- ScyllaDB
-- Cassandra
-- Amazon DynamoDB
-- Python
-- Vertica
-- Graphite
+* Alooma
```
Comment (arikfr) [inline user-guide/about/supported_data_sources.md]: No need to add partners here. None of them isn't really something you query.

File: user-guide/data-sources/bigquery.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# BigQuery Setup
+
+From your Google Cloud Console, you need to create a service account and assign it permissions to access BigQuery. See details on creating a service account here: https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount
```
Comment (arikfr) [inline user-guide/data-sources/bigquery.md]: Should be a link.

File: user-guide/data-sources/bigquery.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+> #### primary::
+>
+> Note that when generating the key file, select the JSON type and later use it when setting > up the data source.
+
+From all the settings the mandatory ones are the key file and the Project ID.
```
Comment (arikfr) [inline user-guide/data-sources/bigquery.md]: This feels like it's related to the service account setup, while in fact it's a comment on our settings...
We can add a screenshot of the settings and explain each one of them. Below it to explain service account creation.

File: user-guide/data-sources/google-analytics.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Google Analytics Instructions
+
+## Setup The Datasource
+### Create a Service Account
+1. Open the [Service accounts page](https://console.developers.google.com/permissions/serviceaccounts). If prompted, select a project.
+2. Click **Create service account**.
+3. In the Create service account window, type a name for the service account, and select Furnish a new private key. When prompted, select **JSON key file type**. Then click **Create**.
```
Comment (arikfr) [inline user-guide/data-sources/google-analytics.md]: hmm looks like a better version of how to create a service account than what we have in bigquery page. let's standartize on one of them and reuse in both places.

File: user-guide/queries/query-google-spreadsheets.md [hunk 6]
```diff
@@ -2,9 +2,12 @@
 
 To load your spreadsheet in Redash share your it with your ServiceAccount’s email (it can be found in the credentials json file, for example 43242343247-fjdfakljr3r2@developer.gserviceaccount.com).
 
+See details on creating a service account [here](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount)
```
Comment (arikfr) [inline user-guide/queries/query-google-spreadsheets.md]: same as before about how to create a service account.

File: user-guide/data-sources/bigquery.md (hunk 4 above)
Comment (afoonalex) [inline user-guide/data-sources/bigquery.md]: Isn't it better to just add a red * to the mandatory fields?

Comment (arikfr) [inline user-guide/data-sources/bigquery.md]: There is one already, but people get confused for some reason :(

**Comment by afoonalex** [comment]
@arikfr can you please review this? I fixed the issues you mentioned, these are useful articles...

File: user-guide/about/supported_data_sources.md (hunk 2 above)
Comment (arikfr) [inline user-guide/about/supported_data_sources.md]: 👆 @afoonalex

**Pull request merged**